capstone-frontend/src/components/BookListAddDropdown.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { DropdownButton, Dropdown } from 'react-bootstrap';
+import CreateList from './CreateList'
+import '../styles/Book.css';
+
+const addBookToBookList = async (bookId, bookListJson) => {
+
+  const bookListUpdateJson = {
+    "id": bookListJson.id,
+    "add_gbookIDs": bookId,
+  }
```

Comment:
nit: Remove the quotes from the field names. You can also remove "Json" from the constant name (it will be written to the request as JSON but it's a plain old JavaScript object now).

Nice use of trailing comma.

Reply:
Fixed!